syntax = "proto2";
package pb;

import "duct.proto";
import "gas.proto";

enum SystemType {
    TaSystem = 0;
}
message System {
    required SystemType systemtype = 1;
    required uint32 Nf = 2;
    required double freq = 3;
    required GasType gastype = 4;
    required double p0 = 5;
    required double T0 = 6;
    map<uint32,Duct> ducts = 7;
    repeated double solution = 8;

}
